content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -28,6 +28,7 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 - The **Add User to Chat** method ([Web](/messaging/agent-chat-api/v3.2/#add-user-to-chat) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#add-user-to-chat)) accepts an optional parameter `require_active_thread`.
 - There's a new parameter `away` in the [Login](/messaging/agent-chat-api/v3.2/rtm-reference/#login) method. It was introduced to prevent chat assignments after unexpected reconnects.
 - Each method, response, and push that deals with a variation of customer `fields` had this field renamed to `session_fields` and its format changed. It's now an array that respects the item order.
+- There're new fields `previous_thread_id` and `next_thread_id` in the [**Chat Thread**](/messaging/agent-chat-api/v3.2/#threads) data structure.
```

Comment:
My only concern is the Chat Thread name. What's our approach to this name? Is it necessary we call is "Chat Thread" instead of just 'Thread"?

Reply:
it can be just `Thread` if we were to assume that Bartek's renames are just around the corner

Reply:
Then I'm for `Thread` 🙌🏻